share/models/validators.py

```diff
@@ -3,29 +3,15 @@
 import ujson
 from collections import OrderedDict
 
-import regex
-
 from jsonschema import exceptions
-from jsonschema import Draft4Validator
+from jsonschema import Draft4Validator, draft4_format_checker
 
 from django.db import connection
 from django.apps import apps
 from django.core.exceptions import ValidationError
 from django.utils.deconstruct import deconstructible
 
-
-def is_valid_uri(value):
-    # uri = rfc3987.get_compiled_pattern('^%(URI)s$')
-    try:
-        assert regex.match(r'.+//:[^/]/+.*', value)
```

Comment:
@aaxelb I think that's the strangest thing we wanted to match. I don't think anything properly formats ISSNs currently, we'll have to keep watch on staging for errors 👍